editor/renames_map_3_to_4.cpp

```diff
@@ -1424,15 +1378,16 @@ const char *RenamesMap3To4::shaders_renames[][2] = {
 };
 
 const char *RenamesMap3To4::class_renames[][2] = {
-	// { "BulletPhysicsDirectBodyState", "BulletPhysicsDirectBodyState3D" }, // Class is not visible in ClassDB.
-	// { "BulletPhysicsServer", "BulletPhysicsServer3D" }, // Class is not visible in ClassDB.
-	// { "GDScriptFunctionState", "Node3D" }, // TODO: Not sure to which this should be changed.
-	// { "GDScriptNativeClass", "Node3D" }, // TODO: Not sure to which this should be changed.
-	// { "InputDefault",""}, // TODO: ?
-	// { "Physics2DDirectBodyStateSW", "GodotPhysicsDirectBodyState2D" }, // Class is not visible in ClassDB.
-	// { "Physics2DShapeQueryResult", "PhysicsShapeQueryResult2D" }, // Class is not visible in ClassDB.
-	// { "PhysicsShapeQueryResult", "PhysicsShapeQueryResult3D" }, // Class is not visible in ClassDB.
-	// { "NativeScript","GDExtension"}, // ??
+	// Too common.
+	// { "Area", "Area3D" },
+	// { "Camera", "Camera3D" },
+	// { "World", "World3D" },
+	// { "Particles", "GPUParticles3D" },
+	// { "Reference", "RefCounted" },
+	// { "Tabs", "TabBar" },
+	// { "Shape", "Shape3D" },
+	// { "Path", "Path3D" },
```

Comment:
This will lead to more manual work for users, as Reference, Area, Camera, Shape and Path are quite used. But those are also potential clashes with user comments and strings.